modules/phase_field/src/postprocessors/GrainTrackerInterface.C

```diff
@@ -33,6 +33,11 @@ validParams<GrainTrackerInterface>()
                         "Terminate with an error if a grain is created "
                         "(does not include initial callback to start simulation)");
 
+  params.addRangeCheckedParam<short>("verbosity_level",
+                                     1,
+                                     "verbosity_level>=0 & verbosity_level<=2",
+                                     "Prints extra information to the console during operation");
+
```

Comment:
Could you explain what information would be printed for `0, 1, 2`, respectively ?